src/config/urls.ts

```diff
@@ -15,3 +15,5 @@ export const STORAGE_URL_PATH = 'asset/v0'
 export const COVER_VIDEO_INFO_URL = 'https://eu-central-1.linodeobjects.com/atlas-hero/cover-info.json'
 
 export const JOYSTREAM_DISCORD_URL = 'https://discord.gg/DE9UN3YpRP'
+
+export const PIONEER_URL = import.meta.env.VITE_PIONEER_URL
```

Comment:
Why aren't you using `readEnv` for that like the rest of URLs

Reply:
Because `redEnv` returns variables for `production`, `development` or `mocking` and I declared `PIONNER_URL` without environment prefix since it's independent of the environment. Maybe we should declare it the same way as we did for discord by adding string to `urls.ts` instead of `.env`?

Reply:
You're right of course. I agree, let's put the URL inside the file itself :+1: